pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+testpaths = unit_test test
```

Comment:
Hmmm, we do need to be careful to not accidentally run all integration tests when trying to run just the unit tests locally during development?
As far as I know, the full tests are meant only for the CI?

Reply:
(coping answer from #213)
Full test can be executed without side effect on linux machine because it use docker and accidenitaly run test on developer machine do not need create destruction, because for proper start the `CIBW_PLATFORM` need to be set.

Reply:
Exactly, running without setting `CIBW_PLATFORM` on macOS (tested) and Windows leads to tests failing with an error message (the same you'd get trying to run cibuildwheel from a local machine).